web_src/css/repo/home-file-list.css

```diff
@@ -71,7 +71,7 @@
 
 #repo-files-table .repo-file-cell.name .entry-name {
   flex-shrink: 1;
-  min-width: 3em;
+  min-width: 4px; /* leave about one letter space when shrinking */
```

Comment:
```suggestion
  min-width: 1ch; /* leave about one letter space when shrinking */
```

https://developer.mozilla.org/en-US/docs/Web/CSS/length#ch

Reply:
<img width="584" height="280" alt="image" src="https://github.com/user-attachments/assets/d5acaad7-f8d2-470e-b0c3-c7f568d9b306" />

<img width="688" height="382" alt="image" src="https://github.com/user-attachments/assets/597ae497-d5d7-4657-9d8f-bad310d80f80" />

Reply:
Hmm yes it's too much, but 4px also seems to much. How about 2px? Effectively we want the width of a space character.

Reply:
Then you won't see the letter

Reply:
Hmm not sure I follow but can't we let it naturally size? Display should be equivalent to `<modname> @ <hash>`, e.g. one space character around the `@`.

Reply:
On narrow screen, then file name part will be shrink-ed to zero then no filename part.

I know it is difficult to fine tune, but the existing space for short names don't look good.

You can try to figure out other better layouts later since this PR majorly aims to "fix"

Reply:
I assume if we can get to render the string like above, it should not shrink like that in limited space. It should just "..." at the end ideally. `min-width: 0` may also fix such shrinkage in a flexbox.